src/cli_utils.rs

```diff
@@ -331,3 +430,12 @@ impl<W: Write> Printer<W> {
         self.print_row()
     }
 }
+
+fn find_char_boundary(s: &str, index: usize) -> usize {
+    for i in 0..4 {
+        if s.is_char_boundary(index + i) {
+            return index + i;
+        }
+    }
+    unreachable!();
+}
```

Comment:
Copying and pasting this feels bad, but the options are either create methods inside of `Report` that are public and should really only be used by CLI version of Tokei or move over the required functionality to not making the library interface more confusing.

I'm not sure which is preferred here, both have their trade offs. I'm willing to move towards whichever is better for long term maintainability and library interface.

Reply:
I think I would be okay with a undocumented re-export.